README.md

```diff
@@ -17,6 +17,8 @@ Learn how to automate your workflow using Twilio's REST API and Twilio SMS. This
 
    Remember that the number where you change the _SMS webhook_ must be the same one you set on the `TwilioPhoneNumber` setting.
 
+   ![Configure Messaging](webhook.png)
+
    To start using `ngrok` in our project you'll have execute to the following line in the _command prompt_.
```

Comment:
People seeing this for the first time will not know what ngrok is or why it is needed. Can we include a small writeup about what it provides and a link to download it? I've noticed this link and summary already exist in other apps, so it can be copy-pasted from there.